sycl/source/detail/device_binary_image.hpp

```diff
@@ -207,6 +210,7 @@ class RTDeviceBinaryImage {
   RTDeviceBinaryImage::PropertyRange ProgramMetadata;
   RTDeviceBinaryImage::PropertyRange ExportedSymbols;
   RTDeviceBinaryImage::PropertyRange DeviceGlobals;
+  RTDeviceBinaryImage::PropertyRange DeviceRequirements;
```

Comment:
This is likely an ABI break. I suggest that we leave RT part (`sycl/`) subdirectory for a separate PR for now to minimize amount of changes which needs to be reviewed

Reply:
Should we alse delete pi.h part for now?

Reply:
Yes, I suggest that we only leave `llvm/` part for now